appengine/standard/endpoints-frameworks-v2/iata/appengine_config.py

```diff
@@ -0,0 +1,4 @@
+from google.appengine.ext import vendor
```

Comment:
needs a license header.

Reply:
This file is identical to (and was copied from) the equivalent file in appengine/standard/endpoints-frameworks-v2/echo.

Reply:
Still needs a license header. :)